lib/src/collection/bimap.dart

```diff
@@ -124,10 +106,9 @@ class HashBiMap<K, V> implements BiMap<K, V> {
   }
 
   @override
-  // TODO: Dart 2.0 requires this method to be implemented.
-  // ignore: override_on_non_overriding_method
   void removeWhere(bool test(K key, V value)) {
-    throw new UnimplementedError("removeWhere");
+    _map.removeWhere(test);
+    _inverse.removeWhere((v, k) => test(k, v));
```

Comment:
nit: Let's do this in the same order as `remove`... not that it really matters :)

Reply:
Done.